seedlet/assets/sass/style.scss

```diff
@@ -69,6 +69,9 @@ Source: https://www.si.edu/object/hymenoxys-odorata-dc:nmnhbotany_13045011
 Included in theme screenshot.
 */
 
+// Variables
+@import "variables";
```

Comment:
For the postcss plugin to work, the variables and styles need to live in the same file.

It made me wonder — why are the front-end variables and stylesheet enqueued separately?

Reply:
> why are the front-end variables and stylesheet enqueued separately?

I was thinking it might be easier for child themes to _replace_ the variables file that way, but it may not be necessary.